mne/viz/_3d.py

```diff
@@ -1490,3 +1625,220 @@ def _get_view_to_display_matrix(scene):
                                  [0.,            0.,   1.,        0.],
                                  [0.,            0.,   0.,        1.]])
     return view_to_disp_mat
+
+
+def _plot_dipole_mri_orthoview(dipole, trans, subject, subjects_dir=None,
+                               coord_frame='head', idx='gof', show_all=True,
+                               ax=None, block=False, show=True):
+    """Plot dipoles on top of mri slices in 3-D.
+
+    Browse through the dipoles using mouse scroll or up/down arrows.
+
+    Parameters
+    ----------
+    dipole : instance of mne.Dipole
+        The dipole to plot.
+    trans : dict | str
+        The mri to head trans or path to the transformation file.
+    subject : str
+        The subject name corresponding to FreeSurfer environment variable
+        SUBJECT.
+    subjects_dir : None | str
+        The path to the freesurfer subjects reconstructions. It corresponds to
+        Freesurfer environment variable SUBJECTS_DIR. If None (default),
+        SUBJECTS_DIR is read from environment or config file.
+    coord_frame : str
+        Coordinate frame to use, 'head' or 'mri'.
+    idx : int | 'gof' | 'amplitude'
+        Index of the initially plotted dipole. Can also be 'gof' to plot the
+        dipole with highest goodness of fit value or 'amplitude' to plot the
+        dipole with the highest amplitude. The dipoles can also be browsed
+        through using up/down arrow keys or mouse scroll. Defaults to 'gof'.
+    show_all : bool
+        Whether to always plot all the dipoles. If True (default), the active
+        dipole is plotted as a red dot and it's location determines the shown
+        MRI slices. The the non-active dipoles are plotted as small blue dots.
+        If False, only the active dipole is plotted.
+    ax : instance of matplotlib Axes3D | None
+        Axes to plot into. If None (default), axes will be created.
+    block : bool
+        Whether to halt program execution until the figure is closed. Defaults
+        to False.
+    show : bool
+        Show figure if True. Defaults to True.
+
+    Returns
+    -------
+    fig : instance of matplotlib figure
+        The figure containing 3-D locations of the dipoles.
+    """
+    import matplotlib.pyplot as plt
+    from mpl_toolkits.mplot3d import Axes3D
+    import scipy
+    from .. import Dipole
+    if has_nibabel():
```

Comment:
the typical pattern is `if not has_nibabel(): raise` and then you don't need to indent the imports or other statements